src/lib/krb5/krb/t_copy_context.c

```diff
@@ -113,7 +112,6 @@ main(int argc, char **argv)
 {
     krb5_context ctx, ctx2;
     krb5_plugin_initvt_fn *mods;
-    const krb5_enctype etypes1[] = { ENCTYPE_DES3_CBC_SHA1, 0 };
```

Comment:
etypes2 should be renamed to etypes or the resulting code will be confusing.

Reply:
Updated.